explorer/nodewatch/nodewatch/NetworkRepository.py

```diff
@@ -192,6 +195,14 @@ def _create_descriptor_from_json(self, json_node):
 		if self._network.generation_hash_seed != Hash256(json_node['networkGenerationHashSeed']):
 			return None
 
+		api_node_info_data = (None, None, None)
+		if 'apiNodeInfo' in json_node:
```

Comment:
i am assuming ``apiNodeInfo`` is getting written by the puller?

Reply:
yes, here is the commit: https://github.com/symbol/miscellaneous/commit/a08027caa22cd70bfe78b2454156326154f35ef4